wallet/src/de/schildbach/wallet/ui/main/WalletActivityExt.kt

```diff
@@ -203,11 +203,6 @@ object WalletActivityExt {
         }
     }
 
-    private val WalletActivity.requestPermissionLauncher: ActivityResultLauncher<String>
-        get() = registerForActivityResult(ActivityResultContracts.RequestPermission()) {
-            requestDisableBatteryOptimisation()
-        }
-
```

Comment:
This construct would create the `ActivityResultLauncher` when it is needed rather than when `WalletActivity` is initialized.

I think this code was generated by pasting Java code into a kotlin file.